src/mpid/ch4/shm/xpmem/xpmem_recv.h

```diff
@@ -9,72 +9,316 @@
 
 #include "ch4_impl.h"
 #include "shm_control.h"
+#include "xpmem_control.h"
 #include "xpmem_pre.h"
 #include "xpmem_seg.h"
 #include "xpmem_impl.h"
 
-/* Handle and complete a matched XPMEM LMT receive request. Input parameters
- * include send buffer info (see definition in MPIDI_SHM_ctrl_xpmem_send_lmt_req_t)
- * and receive request. */
-MPL_STATIC_INLINE_PREFIX int MPIDI_XPMEM_handle_lmt_recv(uint64_t src_offset, uint64_t src_data_sz,
-                                                         uint64_t sreq_ptr, int src_lrank,
-                                                         MPIR_Request * rreq)
+/*
+=== BEGIN_MPI_T_CVAR_INFO_BLOCK ===
+
+cvars:
+    - name        : MPIR_CVAR_CH4_XPMEM_COOP_COPY_CHUNK_SIZE
+      category    : CH4
+      type        : int
+      default     : 32768
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        Chunk size is used in XPMEM COOP P2P and determines the concurrency of
+        copy and overhead of atomic per chunk copy. Best chunk size should
```

Comment:
The CVAR comment is exposed to user. Can you replace `XPMEM COOP P2P` with `XPMEM cooperative copy based point-to-point communication. It determines...`?